packages/tables/src/styled/StyledHeaderCell.ts

```diff
@@ -5,16 +5,73 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled from 'styled-components';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
-import { StyledCell } from './StyledCell';
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { StyledCell, IStyledCellProps } from './StyledCell';
+import { StyledOverflowButton } from './StyledOverflowButton';
 
 const COMPONENT_ID = 'tables.header_cell';
 
+const truncatedStyling = css`
+  ${StyledOverflowButton} {
+    max-width: 100%;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+`;
+
+const sizeStyles = (props: IStyledCellProps & ThemeProps<DefaultTheme>) => {
+  let paddingTop = undefined;
```

Comment:
Fix all padding calculations to be `(rowHeight - lineHeight) / 2`. They are not font-sized based.

Reply:
Updated.